### DISCUSSION TITLE
Disable demo dataset

### DISCUSSION THREAD

**Discussion by Century1026** [inventree/InvenTree#7244] [Q&A] [answered]
Hi,
I initially set up a demo dataset, and now I want an empty dataset. How can I disable the demo dataset?
Looking forward to your answer!

**Comment by SchrodingersGat** [comment, marked as answer]
From here: https://docs.inventree.org/en/stable/start/docker_install/?h=demo+datase#demo-dataset

```
docker compose run --rm inventree-server invoke delete-data
```

*Note: Might be slightly different if you are running bare metal, and not docker*

**Reply by SchrodingersGat** [reply]
You can see all the available invoke commands with `invoke --list`

**Reply by Century1026** [reply]
Thank you so much!
